### ISSUE TITLE
Findings feature request

### ISSUE THREAD

**Issue by Yancruiser** [cobalthq/cobalt-product-public-docs#407]
I really like how the new Findings tab works and easy to download for internal reporting.  Anyway to add a FixedAt 'date' and a conditional when if JIRA integrated, External Issue link columns can be added?

**Comment by JakobStorm** [comment]
Hi! Thank you for suggesting this improvement. Is the FixedAt date something you use to track SLAs or something like that?

Cheers,
Storm

**Comment by Yancruiser** [comment]
Yes, we have internal policy commitments that place timelines to severity levels of the findings.  It is then our goal to meet with those timelines whenever possible and your FixedAt date basically serves as independent validation of remediation.  It is not uncommon for our customers to request evidence that we comply with our own policies and they are please when confirmations are not self-declared, but leverages evidence outside the company.  Hope this helps.
